### DISCUSSION TITLE
Multithreading

### DISCUSSION THREAD

**Discussion by eroltoker** [brefphp/bref#1569] [Support]
When running on a server there are several frameworks to enable multithreading in php.  

Is there any way to achieve this with bref?

**Comment by hschimpf** [comment]
Hey @eroltoker! 

I'm sorry if this reply it's too late, I just saw your question. 

I have a project running PHP with ZTS and `ext-parallel`. You will have to build your custom layer to enable the ZTS flag. 

Check the docs at [Bref: AWS Lambda Layers](https://github.com/brefphp/aws-lambda-layers?tab=readme-ov-file#building) to build your layer. 

Also see this [PR](https://github.com/brefphp/aws-lambda-layers/pull/63) to see how to enable the ZTS flag.

**Reply by eroltoker** [reply]
🙏🙏🙏 Great to know, thank you!
